scripts/policy/frameworks/intel/seen/smtp-url-extraction.zeek

```diff
@@ -21,7 +21,7 @@ event intel_mime_data(f: fa_file, data: string) &group="Intel::URL"
 		}
 	}
 
-event file_new(f: fa_file) &group="Intel::URL"
+event file_new(f: fa_file)
```

Comment:
What's the motivation for this one? Not adding the `ANALYZER_DATA_EVENT` at all seems a useful optimization.

Reply:
I wasn't sure if the DATA_EVENT was useful, and it was causing baseline changes for one of the tests.